src/pages/Calendar/Calendar.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import type { Meta, StoryObj } from "@storybook/react";
```

Comment:
We do not need a story for the Page, stories are for the individual components.

Reply:
file deleted